src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/PopulationStatisticsLoaderStepTest.java

```diff
@@ -108,16 +117,33 @@ public void statisticsLoaderStepShouldLoadStatsIntoDb() throws StorageManagerExc
         assertEquals(1, getCohortStatsFromFirstVariant(cursor).size());
     }
 
-    private Map<String, VariantStats> getCohortStatsFromFirstVariant(DBCursor cursor) {
-        DBObjectToVariantConverter variantConverter = getVariantConverter();
-        Variant variant = variantConverter.convertToDataModelType(cursor.iterator().next());
-        return variant.getSourceEntries().values().iterator().next().getCohortStats();
+    private List<VariantStats> getCohortStatsFromFirstVariant(DBCursor cursor) {
+        assertTrue(cursor.hasNext());
+
+        DBObject dbObject = cursor.iterator().next();
+        VariantDocument variantDocument = mongoOperations.getConverter().read(VariantDocument.class, dbObject);
+        return variantDocument.getVariantStatsMongo().stream()
+                              .map(this::getVariantStats)
+                              .collect(toList());
```

Comment:
Even though I'm not a fan of how streams and lambdas are done in java, I must admit I quite like usages like this one.